api/types/types.go

```diff
@@ -101,6 +101,7 @@ type HuskyCIResults struct {
 	PythonResults     PythonResults     `bson:"pythonresults,omitempty" json:"pythonresults,omitempty"`
 	JavaScriptResults JavaScriptResults `bson:"javascriptresults,omitempty" json:"javascriptresults,omitempty"`
 	RubyResults       RubyResults       `bson:"rubyresults,omitempty" json:"rubyresults,omitempty"`
+	GitResults        GitResults        `bson:"gitresults,omitempty" json:"gitresults,omitempty"`
```

Comment:
What do you think of using `GenericResults`? There may be some generic tools here in the future that may not be related to git and we will need to create a new field for them.